### PR TITLE
Check if nodes has children

### PR DESCRIPTION
**Pull request by sebastiaanspeck**

### CODE REVIEW

File: magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/NestingDepthCheck.java [hunk 1]
```diff
@@ -69,7 +69,16 @@ private void checkNestingDepth(final AstNode node) {
   }
 
   private boolean isEarlyReturn(final AstNode node) {
+    if (!node.hasChildren()) {
+      return false;
+    }
+
     AstNode statement = node.getFirstChild();
+
+    if (!statement.hasChildren()) {
```
Comment (StevenLooman) [inline magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/NestingDepthCheck.java]: Can this happen?

Comment (sebastiaanspeck) [reply magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/NestingDepthCheck.java]: I was in doubt as well.

File: magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/NestingDepthCheck.java [hunk 2]
```diff
@@ -69,7 +69,16 @@ private void checkNestingDepth(final AstNode node) {
...
+      return false;
+    }
+
```
Comment (sebastiaanspeck) [inline magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/NestingDepthCheck.java]:
```suggestion
```

**Comment by StevenLooman** [comment]
Thanks!

**Pull request merged**